packages/package/src/typescript.js

```diff
@@ -28,7 +28,14 @@ export async function emit_dts(input, output, final_output, cwd, alias, files, t
 	const require = createRequire(import.meta.url);
 	const pkg = load_pkg_json(cwd);
 	const svelte_dep = pkg.peerDependencies?.svelte || pkg.dependencies?.svelte || '3.0';
-	const no_svelte_3 = !semver.intersects(svelte_dep, '^3.0.0');
+	let no_svelte_3 = true;
+	try {
+		no_svelte_3 = !semver.intersects(svelte_dep, '^3.0.0');
+	} catch (e) {
+		if (e instanceof Error) {
```

Comment:
is this just to satisfy typescript or is there a case where `e` is not an `Error` where you would like to silently proceed?

Reply:
It's for typescript to have `e.message` just under (line 36).
I'm not aware of other cases